src/content/upgrades/merge/index.md

```diff
@@ -5,64 +5,236 @@ lang: en
+</ExpandableCard>
+
+<ExpandableCard
+title="Misconception: &quot;The Merge will result in downtime of the chain.&quot;"
+contentPreview="False. The Merge upgrade is designed to transition to proof-of-stake with zero downtime.">
+An immense amount of work has been put into making sure the transition to proof-of-stake does not disrupt the network or its users.
+
+The Merge is akin to changing an engine on a rocketship mid-flight, and is designed to be performed without any need to pause anything during the switch. The Merge will be triggered by what is known as a TTD, or [terminal total difficulty](/glossary/#terminal-total-difficult), which is a cumulative measure of the total mining power that has gone into building the chain. When the time comes, and this criteria is met, blocks will go from being produced using proof-of-work in one block, to being produced by proof-of-stake in the next.
+
+Ethereum does not have downtime.
+</ExpandableCard>
 
 ## Relationship between upgrades {#relationship-between-upgrades}
 
 The Ethereum upgrades are all somewhat interrelated. So let’s recap how The Merge relates to the other upgrades.
 
-### The Merge and the Beacon Chain {#docking-and-beacon-chain}
+### The Merge and the Beacon Chain {#merge-and-beacon-chain}
+
+The Merge represents the formal adoption of the Beacon Chain as the new consensus layer to the current Mainnet execution layer. Once The Merge happens, validators will be assigned to secure Ethereum Mainnet and mining [proof-of-work](/developers/docs/consensus-mechanisms/pow/) will no longer be a valid means of block production.
 
-Once The Merge happens, stakers will be assigned to validate the Ethereum Mainnet. [Mining](/developers/docs/consensus-mechanisms/pow/mining/) will no longer be required so miners will likely invest their earnings into staking in the new proof-of-stake system.
+Blocks will instead be proposed by validating nodes that have ether staked for the right to participate in consensus. These upgrades set the stage for future scalability upgrades including data sharding.
 
 <ButtonLink to="/upgrades/beacon-chain/">
   The Beacon Chain
 </ButtonLink>
 
-### The Merge and the post-merge cleanup {#merge-and-post-merge-cleanup}
+### The Merge and the Shanghai upgrade {#merge-and-shanghai}
 
-Immediately after The Merge, some features such as withdrawing staked ETH, will not yet be supported. These are planned for a separate upgrade to follow shortly after The Merge.
+In order to simplify and maximize focus on a successful transition to proof-of-stake, The Merge upgrade will not include certain anticipated features such as the ability to withdraw staked ETH. The Shanghai upgrade is planned to follow The Merge, in which the ability to withdraw is slotted as a top priority.
```

Comment:
```suggestion
In order to simplify and maximize focus on a successful transition to proof-of-stake, The Merge upgrade will not include certain anticipated features such as the ability to withdraw staked ETH. The Shanghai upgrade is planned to follow The Merge, which will enable the ability to stakers to withdraw.
```

> The Shanghai upgrade is planned to follow The Merge, in which the ability to withdraw is slotted as a top priority.

Not a big fan of the wording:
> in which the ability to withdraw is slotted as a top priority.

Can we rephrase this to something more simple?